src/descriptions/id-broker-api.php

```diff
@@ -612,12 +612,59 @@
                 ],
             ],
         ],
+        'emailInternal' => [
+            'httpMethod' => 'POST',
+            'uri' => '/email',
+            'responseModel' => 'Result',
+            'parameters' => [
+                'to_address' => [
+                    'required' => true,
+                    'type' => 'string',
+                    'location' => 'json',
+                ],
+                'cc_address' => [
+                    'required' => false,
+                    'type' => 'string',
+                    'location' => 'json',
+                ],
+                'bcc_address' => [
+                    'required' => false,
+                    'type' => 'string',
+                    'location' => 'json',
+                ],
+                'subject' => [
+                    'required' => true,
+                    'type' => 'string',
+                    'location' => 'json',
+                ],
+                'text_body' => [
```

Comment:
Do we want one of the body elements to be required? If so is there a way to ensure at least one of them is provided?